Source/Redemption/Private/RedemptionController.cpp

```diff
@@ -12,7 +12,7 @@ ARedemptionController::ARedemptionController()
 {
 	PerceptionComp = CreateDefaultSubobject<UAIPerceptionComponent>(TEXT("AiPercption"));
 
-	UAISenseConfig_Sight* SightSenseConfig = CreateDefaultSubobject<UAISenseConfig_Sight>(TEXT("SightSense"));
```

Comment:
Not sure what this change does but I believe in you

Reply:
I needed to change it to be able to change it in the child classes more easily